### DISCUSSION TITLE
New licensing for Sidenav

### DISCUSSION THREAD

**Discussion by sylcha** [mdbootstrap/TW-Elements#2324] [Support from community] [answered]
Hi every one.

I was using Tailwind Elements for a year or so in a project with satisfaction so far until I'm discovering just now that a composant that I used (*Sidenav*) has restricted access now and moved to the pro side... I have no longer access to the API and can't maintain my code anymore.

Such practices are so unfair! One thing is to have restricted work that you have to pay for (every developer need to be paid for their work). Another is to change the rules during a game in progress. I don't understand at all such business decision and I'm really frustrated and disappointed right now. I'll have to rebuild part of my work to avoid using this *Sidenav*. 

I will stop using and recommending  TW Elements anymore.

Sylvain

**Comment by filipkappa** [comment, marked as answer]
Hello Sylvain, 

Apologies for the delay in communication - as you noted we went trough a license model switch, and some instructions & disclaimers are still at works. Please see our announcement on this matter for a full story:
**[TW Elements free is MIT now](https://github.com/mdbootstrap/TW-Elements/discussions/2326)**

Now, regarding your issue with the component.

Regardless of the license change in version 2.0, **everything we released so far will stay open source, and is still a fully functional & reliable tool under AGPL.** 

You can access all versions with their documentation on the same repository [under a specific release tag](https://github.com/mdbootstrap/TW-Elements/releases/).

And here is the full instruction for installation & running the documentation locally:
https://tw-elements.com/docs/standard/getting-started/local-installation/

The documentation is still there, and **you can keep using the components you already used, on the same rules as before**.
The only inconvenience is that you now have to open the documentation locally, because the domain is reserved for a new version.

Once again apologies for the miscommunication, we're trying our best to disclaim all the changes & provide more instructions.
